packages/pyodide-kernel/src/coincident.worker.ts

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Jupyter Development Team.
+// Distributed under the terms of the Modified BSD License.
+
+/**
+ * A WebWorker entrypoint that uses coincident to handle postMessage details
+ */
+import coincident from 'coincident';
+
+import { PyodideRemoteKernel } from './worker';
+import { IPyodideWorkerKernel } from './tokens';
+
+const worker = new PyodideRemoteKernel();
+
+const workerAPI: IPyodideWorkerKernel = coincident(self) as IPyodideWorkerKernel;
+
+workerAPI.initialize = worker.initialize.bind(worker);
```

Comment:
At the very least, we should sort these... but better still would be to: 
- enforce it with a type
- do this dynamically
- automate it with magic comments (ick)

Reply:
@martinRenou is this something that should be done as part of this PR, or could be done as a follow-up?

So we can move forward with this PR and https://github.com/jupyterlite/pyodide-kernel/pull/114

Reply:
We can probably do this as a follow up?